x/common/testutil/cli/logger.go

```diff
@@ -0,0 +1,16 @@
+package cli
```

Comment:
Previously, the `Network`, `Config` (of the network), and `Logger` interface were all defined in one file, which made things hard to follow. I added documentation for parts that weren't obvious, separated these structs to their own files, added tests, and deleted code that was unused in integration tests.